components/application-broker/internal/broker/provision_test.go

```diff
@@ -558,9 +668,23 @@ func TestDoKnativeResourceProvision(t *testing.T) {
 	}
 
 	for _, tc := range []testCase{
+		{
+			name:                           "provision success with new eventing flow",
```

Comment:
I think we need to make sure that Knative resources are not created, for example:
- Check that the namespace was not labeled with `knative-eventing-injection=enabled`
- Check that the Knative subscription was not created.
- Check that the PeerAuthentication was not created.

Reply:
IMHO
- This is already tested since the `expectCreates` and the `initialObjs` check what exactly the objects set is
- knative will be removed soon, it doesn't worth further investment